src/app/persons/person/person.component.ts

```diff
@@ -41,6 +48,7 @@ export class PersonComponent implements OnInit {
         setTimeout(() => {
           this.sendResourceService.setResource(this.person, SentResourceTypes.PERSON_DATA);
```

Comment:
why is this necessary?

Reply:
Because in the addresses-list-component we need to have some info about the person.

Reply:
But this wasn't added in this PR.

Reply:
cool